src/Core/src/PropertyMapper.cs

```diff
@@ -121,6 +132,42 @@ public virtual IEnumerable<string> GetKeys()
 					foreach (var key in chain.GetKeys())
 						yield return key;
 			}
+
+#pragma warning disable CS0612 // Type or member is obsolete
+			if (OverrideMapper is not null)
+			{
+				foreach (var key in OverrideMapper.GetKeys())
+					yield return key;
+			}
+#pragma warning restore CS0612 // Type or member is obsolete
+
+		}
+
+		PropertyMapper? _overrideMapper;
+
+		[Obsolete]
+		internal PropertyMapper? OverrideMapper
+		{
+			get
+			{
+				if (_overrideMapper is not null && _overrideMapper.Keys.Count > 0)
```

Comment:
Only trigger the override mapper if the user has added something to it, otherwise we just fallback to normal behavior